Gruntfile.js

```diff
@@ -66,6 +73,7 @@ module.exports = function (grunt) {
     grunt.loadNpmTasks('grunt-contrib-requirejs');
     grunt.loadNpmTasks('grunt-jsdoc');
     grunt.loadNpmTasks('grunt-karma');
+    grunt.loadNpmTasks('grunt-contrib-copy');
```

Comment:
Nice simple integration. How about moving grunt-contrib-copy next to grunt-contrib-requirejs to establish a clear pattern?